megamek/src/megamek/server/commands/ConvertMMSCommand.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * MegaMek - Copyright (C) 2000-2002 Ben Mazur (bmazur@sev.org)
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the License, or (at your option)
+ *  any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ *  for more details.
+ */
+
+/*
+ * ConvertMMSCommand.java
+ *
+ * Created on Jan 10, 2020
+ * Version 0.2
+ */
+
+package megamek.server.commands;
```

Comment:
Double spacing is an accessibility issue, and thus these should be removed. Further, it would be best to put this under Copyright for the MegaMek Team instead of Ben Mazur. Finally, the other comment should be removed and the information merged with the Javadoc comment that starts the class.